kbase-extension/static/kbase/js/common/errorDisplay.js

```diff
@@ -23,17 +23,25 @@ define(['bluebird', 'common/html'], (Promise, html) => {
      */
 
     function normaliseErrorObject(rawObject) {
-        if (rawObject.exec && rawObject.exec.jobState) {
-            rawObject = { jobState: rawObject.exec.jobState };
-        }
-
         let errorObject;
-        if (rawObject.jobState && (rawObject.jobState.error || rawObject.jobState.errormsg)) {
-            errorObject = convertJobError(rawObject.jobState);
+        if (rawObject.fatalError) {
+            errorObject = convertInternalError(rawObject.fatalError);
```

Comment:
the error display module was ignoring these beautiful fatalErrors! How very dare it.